skimage/color/colorconv.py

```diff
@@ -1180,7 +1189,13 @@ def lab2rgb(lab, illuminant="D65", observer="2", *, channel_axis=-1):
     ----------
     .. [1] https://en.wikipedia.org/wiki/Standard_illuminant
     """
-    return xyz2rgb(lab2xyz(lab, illuminant, observer))
+    xyz, invalid = _lab2xyz(lab, illuminant, observer)
+    if len(invalid) != 0:
+        warn('Color conversion from LAB to RGB via negative Z values. '
+             'Those values have been clipped to zero.',
+             stacklevel=3)
+
+    return xyz2rgb(xyz)
```

Comment:
Side question, channel_axis seem to be unused, is that on purpose ?

Reply:
It is being handled via the `@channel_as_last_axis()` decorator which will move the specified channel axis to the last position as assumed by the implementation.

Reply:
So ... it does not need to be a parameter of this function, and is there just for the signature.

Reply:
As far as I'm aware, I think you are correct. It's not needed at runtime but at the very least necessary for `help(...)` to show the signature properly.